aopy/data/db.py

```diff
@@ -452,15 +452,16 @@ def get_preprocessed_sources(self):
         Returns:
             list: preprocessed sources for this task entry
         '''
-        params = self.task_params
         sources = ['exp']
+        if 'quatt_bmi' in self.features:
+            sources.append('emg')
+        params = self.task_params
         if 'record_headstage' in params and params['record_headstage']:
             sources.append('broadband')
             sources.append('lfp')
         if 'neuropixels' in self.features:
             sources.append('spike')
             sources.append('lfp')
-        sources.append('eye')
```

Comment:
we probably shouldn't delete this but i'm not sure how to fix it because there isn't a feature for oculomatic..